Mailman tests for existence of users during configure -- unfortunately
this creates a problem; we create the users just before install.

We remove the testing part.

--- configure.in.bak	2010-09-20 19:18:27.000000000 +0100
+++ configure.in	2011-06-14 20:20:29.209438957 +0100
@@ -311,28 +311,7 @@
 # $2 == user id to check for
 AC_SUBST($1)
 changequote(,)
-if test -z "$$1"
-then
-    cat > conftest.py <<EOF
-import grp
-gid = ''
-for group in "$2".split():
-    try:
-        try:
-            gname = grp.getgrgid(int(group))[0]
-            break
-        except ValueError:
-            gname = grp.getgrnam(group)[0]
-            break
-    except KeyError:
-        gname = ''
-fp = open("conftest.out", "w")
-fp.write("%s\n" % gname)
-fp.close()
-EOF
-    $PYTHON conftest.py
-    $1=`cat conftest.out`
-fi
+$1=$2
 changequote([, ])
 rm -f conftest.out conftest.py])
 
@@ -343,28 +322,7 @@
 # $2 == user id to check for
 AC_SUBST($1)
 changequote(,)
-if test -z "$$1"
-then
-    cat > conftest.py <<EOF
-import pwd
-uid = ''
-for user in "$2".split():
-    try:
-        try:
-            uname = pwd.getpwuid(int(user))[0]
-            break
-        except ValueError:
-            uname = pwd.getpwnam(user)[0]
-            break
-    except KeyError:
-        uname = ''
-fp = open("conftest.out", "w")
-fp.write("%s\n" % uname)
-fp.close()
-EOF
-    $PYTHON conftest.py
-    $1=`cat conftest.out`
-fi
+$1=$2
 changequote([, ])
 rm -f conftest.out conftest.py])
 
